example/connectors.ts

```diff
@@ -26,14 +26,15 @@ export const network = new NetworkConnector({
 })
 
 export const walletconnect = new WalletConnectConnector({
-  rpc: { 1: RPC_URLS[1], 4: RPC_URLS[4] },
```

Comment:
example app was throwing an error about WalletConnect not liking multiple rpc url's.